package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "guetzli",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
Version bumps should be done in a separate commit. I would recommend using https://github.com/sindresorhus/np to make releases safely.